Dockerfile

```diff
@@ -23,13 +23,12 @@ ADD package.json /src/package.json
 ADD bower.json /src/bower.json
 ADD .bowerrc /src/.bowerrc
 
-RUN cd /src && npm install bower
-RUN cd /src && npm install --production --unsafe-perm
-
 ADD . /src
 
 ADD config/db.json.docker /src/config/db.json
 
+RUN npm install -g mocha grunt-cli bower karma-cli && cd /src && npm install --unsafe-perm
```

Comment:
We were installing only production deps with `--production`, can you restore it please ?